### PR TITLE
Authorize raw eeBUS MCP operator boundary

### PR DESCRIPTION
**Pull request by d3vi1**
## Summary
- add strict RED contract tests for the public redacted and owner-only raw eeBUS MCP boundaries
- add the canonical VR940-shaped raw fixture with 1 device, 11 entities, 20 features, and 10 use-case claims
- preserve the closed nine-tool namespace while testing boundary-bound references, Unix peer authorization, secret exclusion, and deterministic immutable snapshots

## TDD state
RED only. No production code or dependency changes are included. The new tests intentionally fail until the #743 GREEN implementation is added.

## Validation
- baseline before changes: `GOWORK=off go test ./mcp ./cmd/gateway` passed
- focused RED: `GOWORK=off go test ./mcp ./cmd/gateway -run TestIssue743 -count=1` fails on the missing operator boundary, old public auth scope, and pre-v0.1.15 dependency
- full RED: `GOWORK=off go test ./...` fails only in `mcp` and `cmd/gateway` on the #743 contract

Closes #743

### CODE REVIEW

File: mcp/eebus_v1.go [hunk 1]
```diff
@@ -52,6 +52,37 @@ type eebusV1Runtime struct {
...
+	AuthScope             string
+}
+
+var (
+	eebusV1PublicBoundary = eebusV1Boundary{
+		AuthorizationBoundary: "lan-http",
+		MaskTier:              "redacted",
+		AuthScope:             "eebus.public.read",
+	}
+	eebusV1OperatorBoundary = eebusV1Boundary{
```
[the remark this answers is not in the document]
Comment (d3vi1) [reply mcp/eebus_v1.go:71]: Resolved by design: the initial eebus.v1 contract has not been released or adopted by consumers. The operator explicitly required correcting this single unpublished namespace in place, with no v2, legacy aliases, or compatibility surface; canonical docs PR69/PR71 record that decision.

File: mcp/eebus_v1_store.go [hunk 2]
```diff
@@ -132,9 +135,17 @@ func newEEBusV1SnapshotStore(now func() time.Time, entropy io.Reader) *eebusV1Sn
...
+func (store *eebusV1SnapshotStore) capture(projection eebusV1Projection, boundaries ...eebusV1Boundary) (eebusV1CapturedRootV1, string) {
 	store.mu.Lock()
 	defer store.mu.Unlock()
+	boundary := projection.Boundary
+	if len(boundaries) != 0 {
+		boundary = boundaries[0]
+	}
+	if boundary != projection.Boundary ||
+		(boundary != eebusV1PublicBoundary && boundary != eebusV1OperatorBoundary) {
+		return eebusV1CapturedRootV1{}, "permission_denied"
```
[the remark this answers is not in the document]
Comment (d3vi1) [reply mcp/eebus_v1_store.go:152]: Accepted. I am partitioning capture quota by authorization boundary so LAN redacted captures cannot exhaust the owner raw budget; regression coverage will prove independent quotas.

[the remark this answers is not in the document]
Comment (d3vi1) [reply mcp/eebus_v1_store.go:152]: Fixed in d73663cb8752e983b3f661ff29118b1b3fb8323a. Active roots now carry an explicit authorization boundary and the 32-root quota is counted independently for public redacted and operator raw captures. Tombstone retention/eviction is also bounded independently at 256 per boundary, with deterministic decoded-root ordering inside each partition. Regression evidence: TestIssue743CaptureQuotaIsIndependentAndExactPerBoundary, TestIssue743FullCaptureQuotaCannotStarveOtherBoundary, and TestIssue743TombstonesAreBoundedPerBoundaryByDecodedRootOrdering. Local focused race, full go test/race/vet/lint/ci_local all passed; Linux CI run 30210861161 job 89816719912 passed.

File: mcp/eebus_v1_dto.go [hunk 3]
```diff
@@ -128,734 +69,128 @@ type eebusV1SnapshotMetaDataV1 struct {
...
-	runtimeKey string
-}
-
-func newEEBusV1Pseudonymizer(key []byte, runtime eebusV1SourceIdentity) (*eebusV1Pseudonymizer, error) {
-	if len(key) != sha256.Size {
-		return nil, errors.New("pseudonym key must contain 32 bytes")
-	}
-	if err := eebusV1ValidateSourceIdentity(runtime); err != nil {
-		return nil, fmt.Errorf("runtime identity: %w", err)
+func eebusV1ProjectSnapshot(value any, _ []byte) (eebusV1Projection, error) {
```
[the remark this answers is not in the document]
Comment (d3vi1) [reply mcp/eebus_v1_dto.go]: Fixed in d73663cb8752e983b3f661ff29118b1b3fb8323a. Public/shareable IDs are now HMAC-SHA256 pseudonyms over the irreversible eebusreg redacted projection, keyed by the process-ephemeral key and domain-separated for runtime/service/session/remote/device/entity/feature/usecase. Collections and the public snapshot hash are canonicalized from keyed IDs; dependency-stable correlators and keys never enter output. Raw operator projection remains unchanged, and synchronized evidence uses its generated key. Regression evidence: TestIssue743PublicPseudonymsAreKeyedEphemeralAndDomainSeparated, TestIssue743RawOperatorProjectionIgnoresPublicPseudonymKey, and TestIssue743SynchronizedEvidencePublicIDsUseProvidedEphemeralKey. Local focused race, full go test/race/vet/lint/ci_local all passed; Linux CI run 30210861161 job 89816719912 passed.

File: mcp/eebus_v1.go [hunk 4]
```diff
@@ -220,36 +259,39 @@ func eebusV1Spec(name string) (eebusV1ToolSpec, bool) {
...
-		return runtime.envelopeAt(spec, "evidence", timestamp, status, data, nil)
+		return runtime.envelopeAt(spec, "evidence", timestamp, status, boundary, data, nil)
 	}
 
-	return runtime.liveCall(ctx, spec, validated)
+	return runtime.liveCall(ctx, spec, validated, boundary)
 }
 
-func (runtime *eebusV1Runtime) liveCall(ctx context.Context, spec eebusV1ToolSpec, validated eebusV1ValidatedArguments) eebusV1EnvelopeV1 {
+func (runtime *eebusV1Runtime) liveCall(ctx context.Context, spec eebusV1ToolSpec, validated eebusV1ValidatedArguments, boundary eebusV1Boundary) eebusV1EnvelopeV1 {
```
[the remark this answers is not in the document]
Comment (d3vi1) [reply mcp/eebus_v1.go:296]: Fixed in d73663cb8752e983b3f661ff29118b1b3fb8323a. Live calls now use independent bounded eight-slot worker semaphores for public LAN and operator AF_UNIX boundaries. The budget is selected from the server-injected boundary; spoofed headers remain public and cannot consume operator capacity. Regression evidence: TestIssue743SaturatedPublicWorkersCannotStarveOperatorBoundary deterministically blocks all eight public provider calls, verifies a spoofed public request times out as redacted, then verifies the operator raw call enters as provider call nine and succeeds. The test passed repeatedly and under race; full local gates and Linux CI run 30210861161 job 89816719912 passed.
